### PR TITLE
Slabtext added and styled colors

### PR DESCRIPTION
**Pull request by johnchowork**
Colours have been reviewed to a softer tone
Added another colour step at 10 bikes left
Added title
Added SlabText (http://freqdec.github.io/slabText/)

### CODE REVIEW

File: wallboard.js [hunk 1]
```diff
@@ -48,10 +48,15 @@ app.use(function *(){
 	}
 
 	var station = response.body[0];
-	var color = 'green';
-	if (station.bikes <= 5) {
-		color = 'red';
-	}
+	var color = '#5cb85c';
+
+    //We should use percentages based on total number of stations
```
Comment (euoia) [inline wallboard.js]: I think the indentation is a bit wonky here. Can you fix that please?

Comment (euoia) [inline wallboard.js:52]: The indentation looks a bit wonky here. Can you fix that please?

**Comment by euoia** [comment]
The indentation is still using mixed tabs and spaces. Check this screenshot from vim with `:set list` enabled showing tabs as `>---`.

image

**Comment by euoia** [comment]
I tested this and it looks good but on my laptop screen the horizontal center alignment seems to be off a bit on the `<h1>{{bikes}}</h1>`. Maybe the text is a bit too big?

**Comment by johnchowork** [comment]
I will place a max-font-size on it, I will test and I can imagine that it will resolve that issue.
I'll also fix the indentation issue, I just need to edit my .vimrc first.

**Pull request merged**